.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Please don't commit this file.  Or `dist/ramda.js`.  Or `dist/ramda.min.js`.

Reply:
My sincere apologies. I believe I have remedied the problem - please do let me know if I have not done so correctly, particularly with the files in `dist/`.

Reply:
No problem at all.  We really need to improve our process.  However, I think you didn't get it right.  Now the commit is deleting the files.  We probably shouldn't have that.  We want these in the release but not every commit.  After we get this fix in, I'm planning on making a new release, and I'll try to fix things so that these two files are in a .gitignore.  I'm not quite sure why they are not there now.

Reply:
I'm afraid I might have mucked this branch so that I'm unable to fix the problem with `dist/`. I have created a new branch `is-bugfix-3196`. I have confirmed that the folder is correctly represented there. Perhaps it would be best to initiate the pull request from that branch?